### PR TITLE
Add two new glossary terms

### PR DESCRIPTION
**Pull request by djgarr**
Adds glossary terms for "Cooperatives" and "Single candidate independent expenditure filer" (refer to #4793 (comment) for definitions)

Resolves #/fec-cms/issues/4793

### CODE REVIEW

File: fec_eregs/static/fec_eregs/js/data/terms.json [hunk 1]
```diff
@@ -103,6 +103,10 @@
...
   },
+  {
+    "term": "Cooperatives",
+    "definition": "A cooperative is a voluntary economic association organized to provide economic services, without gain, to itself, and to the members who own and control it. A federated cooperative as defined in the Agricultural Marketing Act of 1929, 12 U.S.C. 1141j, or a rural cooperative eligible for assistance under chapter 31 or title 7 of the United States Code, may solicit the members of the cooperative's regional, state or local affiliates, provided that all of the political committees established, financed, maintained or controlled by the cooperative and its regional, State or local affiliates are considered one political committee for the purposes of the limitations in 11 CFR <a href=\"https://www.fec.gov/regulations/110-1/CURRENT#110-1\">110.1</a> and <a href=\"https://www.fec.gov/regulations/110-2/CURRENT#110-2\">110.2</a>."
+  },
```
Comment (dorothyyeager) [inline fec_eregs/static/fec_eregs/js/data/terms.json:109]: Need to make State lower case s.

**Review by dorothyyeager — CHANGES_REQUESTED** [review]
One minor change to make and then I can approve @djgarr. Thanks!

File: fec_eregs/static/fec_eregs/js/data/terms.json (hunk 1 above)
Comment (dorothyyeager) [reply fec_eregs/static/fec_eregs/js/data/terms.json:109]: Actually a question too - should it be Cooperative (singular) or Cooperatives (plural)? Looking at other glossary terms, I think it should be singular with no s on the end unless there's a design reason that @rfultz and @JonellaCulmer want it to be plural.

Comment (djgarr) [reply fec_eregs/static/fec_eregs/js/data/terms.json:109]: Changes made

Comment (dorothyyeager) [reply fec_eregs/static/fec_eregs/js/data/terms.json:109]: I'd like to take the s out of cooperative too, I think.

Comment (djgarr) [reply fec_eregs/static/fec_eregs/js/data/terms.json:109]: S removed from cooperative

**Review by dorothyyeager — APPROVED** [review]
Looks good. Don't forget to update the spreadsheet and the list we keep in the google drive. Thanks for bearing with this!

**Pull request merged**